Update Invoke-ControlCommand.ps1

DaveidWeba opened the pull request:
Fixes issue #75. Thanks to @higherinfogroup for sharing the remedy.

DarrenWhite99 commented:
So that I can accept this push and get all the modifications in one step (lazy right?) can you update your AutomateAPI.psd1 to nudge the module version.
Change 
ModuleVersion = '1.1.5'
to
ModuleVersion = '1.1.6'
... and then I'll pull the branch over to master here.
Thanks

DaveidWeba commented:
@DarrenWhite99 Apologies for the delay, life gets busy! Just updated the ModuleVersion to 1.1.6 as requested. Also updated the Tested Versions of the ReadMe to my environment's versions of Automate and ScreenConnect.
